packages/radix-vue/src/Toast/ToastRootImpl.vue

```diff
@@ -79,7 +79,7 @@ const remainingRaf = useRafFn(() => {
 }, { fpsLimit: 60 })
 
 function startTimer(duration: number) {
-  if (!duration || duration === Number.POSITIVE_INFINITY)
+  if (duration <= 0 || duration === Number.POSITIVE_INFINITY)
```

Comment:
```suggestion
  if (!duration || duration <= 0 || duration === Number.POSITIVE_INFINITY)
```

In case the duration value is `undefined`